tasks/1_golf-score/golf-score.js

```diff
@@ -0,0 +1,19 @@
+export const GOLF_SCORES = {
+  HOLE_IN_ONE: `Hole_in_one`,
+  EAGLE: `Eagle`,
+  BIRDIE: `Birdie`,
+  PAR: `Par`,
+  BOGEY: `Bogey`,
+  DOUBLE_BOGEY: `Double Bogey`,
+  GO_HOME: `Go Home`,
+};
+
+export function golfScore(par, strokes) {
+  if (strokes == 1) return GOLF_SCORES.HOLE_IN_ONE;
+  else if (strokes <= par - 2) return GOLF_SCORES.EAGLE;
+  else if (strokes == par - 1) return GOLF_SCORES.BIRDIE;
+  else if (strokes == par) return GOLF_SCORES.PAR;
+  else if (strokes == par + 1) return GOLF_SCORES.BOGEY;
+  else if (strokes == par + 2) return GOLF_SCORES.DOUBLE_BOGEY;
+  else if (strokes >= par + 3) return GOLF_SCORES.GO_HOME;
+}
```

Comment:
another approach will be use switch and count difference only once

Reply:
1. use === instead of == for comparations.
2. brackets are always good practise. imagine that someone opened your code in editor with diffrent formmating, and then editor did some code formmating. without brackets code is harder to read and maintain